components/jobsBoard.js

```diff
@@ -0,0 +1,134 @@
+import Link from "next/link";
+import Button from "components/button";
+import Paragraph from "components/paragraph";
+
+import IconComponent from "components/icons";
+
+const JobsBoard = ({ jobs }) => {
+  return (
+    <div
+      id="jobs"
+      className="container p-4 justify-center"
+    >
+      <h1
+        className="com-head text-center"
+        data-aos="fade-down"
+        data-aos-duration="800"
+      >
+        Jobs
+      </h1>
+      <div className="my-4 text-center">
+        <p>
+          Welcome to the PyCon India 2023 Job Board – your gateway to exciting career opportunities in the world of Python! PyCon India is one of the largest and most vibrant Python conferences in the Asia-Pacific region, and this year, we are thrilled to offer an exclusive platform for Python enthusiasts and companies to connect, network, and explore career possibilities.
+        </p>
+        <Link
+          href="jobs/rules-guidelines"
+          target="_blank"
+        >
+          Rules and Guidelines
+        </Link>
+        <br />
+        <Button
+          buttonClassName="custom-button green-btn px-5 my-4"
+          anchorClassName="text-decoration-none text-light"
+          buttonLabel="Submit a Job"
+          buttonHyperLink="https://forms.gle/1b1VeRprtCUMcdJs9"
+        />
+      </div>
+      <div className="row my-4">
+        <div className="text-center">
+          {Array.isArray(jobs) && jobs.length === 0 &&
+            <h3
+              className="text-center"
+              data-aos="fade-down"
+              data-aos-duration="800"
+            >Currently No Job Listings Available – Check Back Soon!
+            </h3>
+          }
+        </div>
+        {jobs && jobs.map((job, index) => (
```

Comment:
we should add some form of pagination after a certain limit

Reply:
I haven't implemented pagination, as I anticipate that there won't be a large number of jobs.
We can add it later if the volume of jobs increases.